src/pyfaf/storage/problem.py

```diff
@@ -133,44 +136,44 @@ def quality(self):
         return self.sorted_reports[0].quality
 
     @property
-    def sorted_reports(self):
+    def sorted_reports(self) -> List[Any]:
         '''
         List of all reports assigned to this problem
         sorted by quality.
         '''
         return sorted(self.reports, key=lambda r: r.quality, reverse=True)
 
     @property
-    def active_reports(self):
+    def active_reports(self) -> List[Dict[str, Any]]:
         '''
         List of all non archived reports
         '''
         return [r for r in self.reports if not r.archived]
 
     @property
-    def archived_reports(self):
+    def archived_reports(self) -> List[Any]:
         '''
         List of archived reports
         '''
         return [r for r in self.reports if r.archived]
 
     @property
-    def backtraces(self):
+    def backtraces(self) -> List[Report]:
```

Comment:
How did you come to this conclusion?

Reply:
Shouldn’t this be `List[ReportBacktrace]`?

https://github.com/abrt/faf/blob/8b5af92f8915f15b8049822e397a5573ee901041/src/pyfaf/storage/report.py#L166